armi/reactor/converters/tests/test_axialExpansionChanger.py

```diff
@@ -634,6 +634,12 @@ def test_isFuelLocked(self):
             the_exception = cm.exception
             self.assertEqual(the_exception.error_code, 3)
 
+    def test_determineLinked(self):
```

Comment:
Can we also add one more `assert` here that tests what happens if one `Component` is unshaped but one _has_ a shape?

It's just nice to test all the permutations.

And if you're thinking "but that's not a concern with _my_ code", consider it future-proofing for 4 years from now when someone _else_ makes a change to this feature.  :)

Reply:
Addressed https://github.com/terrapower/armi/pull/778/commits/6438d7540afd2e7ccb839d2c7b7735a91946f813